test/gtest/ucs/test_bitmap.cc

```diff
@@ -5,12 +5,13 @@
  */
 
 #include <common/test.h>
-
 #include <ucs/datastruct/bitmap.h>
 
-class test_ucs_bitmap : public ucs::test {
+class test_ucs_bitmap : public ucs::test
+{
```

Comment:
Should change in this or another PR:
* BraceWrapping:
  AfterClass: ~~true~~false
* Add UCS_BITMAP_FOR_EACH_BIT to ForEachMacros

Reply:
In another PR - this branch doesn't have clang format.